### PR TITLE
Updated Devfest Vizag details

### PR DESCRIPTION
**Pull request by CharanMN7**
- Pulled recent changes from repo to avoid conflicts while merging.
- Added the event link.
- Added the description.

### CODE REVIEW

**Review by sampathbalivada — APPROVED** [review]
LGTM

File: data/events.json [hunk 1]
```diff
@@ -121,8 +121,8 @@
     "StartingDate": "2023-12-02",
     "EndingDate": "2023-12-02",
     "Venue": "TBD (Visakhapatnam, India)",
```
Comment (sampathbalivada) [inline data/events.json]: @CharanMN7 please update the Venue as "GITAM University, Visakhapatnam"

**Review by sampathbalivada — CHANGES_REQUESTED** [review]
Requested changes

File: data/events.json (hunk 1 above)
Comment (CharanMN7) [reply data/events.json]: updated the venue to "GITAM University, Visakhapatnam" 👍

**Review by sampathbalivada — APPROVED** [review]
LGTM

**Comment by sampathbalivada** [comment]
@vrijraj Could you please merge these changes? Thanks.

**Comment by priyaljain919** [comment]
@sampathbalivada @vrijraj done

**Pull request merged**